drivers/hmis/lib/form_data/default/fragments/prior_living_situation_b.json

```diff
@@ -229,7 +229,7 @@
         "record_type": "ENROLLMENT",
         "field_name": "previousStreetEssh"
       },
-      "text": "On the night before project entry, did the client stay on the streets, ES or SH?",
+      "text": "On the night before, did the client stay on the streets, ES or SH?",
```

Comment:
@eanders I reverted this recent change to the original wording for now https://www.pivotaltracker.com/story/show/186701809/comments/239633121